.circleci/config.yml

```diff
@@ -66,7 +66,7 @@ jobs:
           command: |
             yarn remove vue vue-jest vue-template-compiler babel-preset-es2015
             yarn add vue@3.0.6 vue-jest@5.0.0-alpha.10 vue-loader@16.1.2 jest@26.6.3 babel-jest@25.2.6 @babel/preset-env@7.14.5 -D -E
-            cp ./scripts/vue-utils-v3.js src/util/vue-utils.js
+            echo "export * from './index-3';" > src/util/vue/index.js
```

Comment:
why doesn't this use yarn build:vue-3 or the build-vue3.sh script?

Reply:
When we run unit tests, we actually don't need `dist`.

Reply:
I see, would still be interesting to be able to test vue 3 locally without swapping dependencies though

Reply:
Yeah the local workflow to fix a failing unit test in vue 3 is quite a mess. So far I have no idea.